src/shared/orgListUtil.ts

```diff
@@ -129,6 +129,10 @@ export class OrgListUtil {
    * @param fileNames All the filenames in the global hidden folder
    */
   public static async readAuthFiles(fileNames: string[]): Promise<AuthInfo[]> {
+    // Ensure that the Global.SFDX_DIR exists
+    // https://github.com/forcedotcom/cli/issues/2222
+    await fs.mkdir(Global.SFDX_DIR, { recursive: true });
```

Comment:
this'll throw if the directory already exists though, do we need to wrap this in try/catch?

Reply:
Oh! I didn't look closely at what you were testing. The `mkdir` function will throw if you are not passing `{ recursive: true }` ([docs](https://nodejs.org/api/fs.html#fspromisesmkdirpath-options:~:text=Calling%20fsPromises.mkdir()%20when%20path%20is%20a%20directory%20that%20exists%20results%20in%20a%20rejection%20only%20when%20recursive%20is%20false)). I just tested this code several more times and it is working as expected.